aiidalab_widgets_base/utils/loaders.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+from IPython.display import Javascript, display
+
+
+def load_css_stylesheet(css_dir: Path, filename: str = ""):
```

Comment:
Thank you, this looks better! 

...now that we're passing a path, I am wondering, it would seem more natural to have a single parameter, that can be either a dir or a file. The split to two parameters for dir and file doesn't seem intuitive, and I don't see any advantage of doing it that way. 

(Sorry, I will shut up eventually : 😅) But perhaps good to iterate on this interface now rather to make it as intuitive as possible).

Reply:
I'd prefer to keep the dir and filename separate. The reason for it is clear in the logic of the function.

Reply:
Not sure if the function logic should define its interface. From a users perspective, passing a single path seems such simpler to think about (also it's not great that the two parameters have different type, another thing that can trip the user).

@superstar WDYT? 

(otherwise LGTM, thanks!)

Reply:
Sorry, I guess I was replying to the older version of your comment. 

Yes, I would expect the the function can internally use Path.is_dir() to determine what to do.

Otherwise I am happy with the changes.

Reply:
Your comments regarding the API are valid. In general, the user comes first (reasonably). I'll adjust the code in the morning. Thanks.

Reply:
API now takes `css_path` of type `Path` or `str` (the latter converted to `Path`).

Reply:
Looks great, thank you!